beaconclient/multi_beacon_client.go

```diff
@@ -64,6 +76,7 @@ type MultiBeaconClient struct {
 
 	// feature flags
 	ffAllowSyncingBeaconNode bool
+	ffBroadcastValidation    BroadcastValidation
```

Comment:
`ff` prefix means feature flag, usually bool 

this should probably be `broadcastMode`